### PR TITLE
Merge master into 2.0-dev

### PR DESCRIPTION
**Pull request by wilsonge**
Pull Request to watch travis pass as conflicts were hard

### CODE REVIEW

File: Tests/Cases/SqlsrvCase.php [hunk 1]
```diff
@@ -104,7 +104,17 @@ protected function getConnection()
 
 		// Create the PDO object from the DSN and options.
 		$pdo = new \PDO($dsn, static::$options['user'], static::$options['password']);
-		$pdo->exec('create table [dbtest]([id] [int] IDENTITY(1,1) NOT NULL, [title] [nvarchar](50) NOT NULL, [start_date] [datetime] NOT NULL, [description] [nvarchar](max) NOT NULL, CONSTRAINT [PK_jos_dbtest_id] PRIMARY KEY CLUSTERED ([id] ASC) WITH (STATISTICS_NORECOMPUTE  = OFF, IGNORE_DUP_KEY = OFF))');
+        $pdo->exec("
```
Comment (mbabker) [inline Tests/Cases/SqlsrvCase.php]: PHPCS

File: Tests/DatabaseDriverTest.php [hunk 2]
```diff
@@ -362,6 +362,28 @@ public function testQuoteFloat()
...
 	}
 
+    /**
+     * Tests the Joomla\Database\DatabaseDriver::quoteBinary method.
+     *
+     * @return  void
+     *
+     * @since   1.7.0
+     */
+    public function testQuoteBinary()
```
Comment (mbabker) [inline Tests/DatabaseDriverTest.php]: PHPCS on the whole method

**Comment by mbabker** [comment]
You lost a bit of https://github.com/joomla-framework/database/commit/97ae8104df7e32bc3ba9386b70eecede062ae466 (PHP 7.3 on AppVeyor)

**Comment by wilsonge** [comment]
7.3 is there - it's just a single line

**Comment by mbabker** [comment]
It's a single line on master but it's not there at all on your branch (even looked at https://github.com/joomla-framework/database/blob/2625f1f30db6a249c200f8a0f221e9e120dc46a8/.appveyor.yml to make sure I didn't FUBAR the diff)

**Comment by mbabker** [comment]
Other than that, make sure `quoteBinary` and `decodeBinary` hit the interface at some point and on a glance everything looks G2G.

**Comment by wilsonge** [comment]
Cool. I just accidentally pushed the commit that added them into the interface. i guess we just fix it as we go now 😆

**Pull request merged**